src/freeclimb/Model/CreateConferenceAllOf.cs

```diff
@@ -211,12 +210,12 @@ public override int GetHashCode()
                     hashCode = (hashCode * 59) + this.ActionUrl.GetHashCode();
                 }
                 hashCode = (hashCode * 59) + this.Alias.GetHashCode();
-                if (this.PlayBeep != null)
+                hashCode = (hashCode * 59) + this.PlayBeep.GetHashCode();
+                hashCode = (hashCode * 59) + this.Record.GetHashCode();
+                if (this.StatusCallbackUrl != null)
                 {
-                    hashCode = (hashCode * 59) + this.PlayBeep.GetHashCode();
+                    hashCode = (hashCode * 59) + this.StatusCallbackUrl.GetHashCode();
                 }
-                hashCode = (hashCode * 59) + this.Record.GetHashCode();
-                hashCode = (hashCode * 59) + this.StatusCallbackUrl.GetHashCode();
                 if (this.WaitUrl != null)
                 {
```

Comment:
General comment - how is it only the create conference models were the only ones effected? I'm pretty sure a lot more models got hit with the enum updates.

Reply:
That's a really good question, not sure, I did an auto-generate of the files and this is what happened, maybe we might need to get on a call to discuss this

Reply:
I believe this comment can be resolved since it was proved that models are up to date with enums